Dockerfile

```diff
@@ -35,6 +35,10 @@ RUN mkdir -p /var/www/$appname \
     && chown nginx -R /var/www/.cache/Python-Eggs/ \
     && chown nginx /var/www/$appname
 
+# py https wants to access $HOME/.netrc -
```

Comment:
when does this happen?

Reply:
the new authlib uses `httpx` - I mistyped that as https